src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -460,6 +462,23 @@ class K8sServiceImpl implements K8sService {
         }
     }
 
+    /**
+     * Fetch current available logs of a running pod
+     *
+     * @param name The pod name
+     * @return The logs as a string or when logs are not available or cannot be accessed
+     */
+    @Override
+    String getCurrentLogsPod(String name) {
+        try {
+            return k8sClient.coreV1Api().readNamespacedPodLog(name, namespace, null, null, null, null, null, null, null, null, null)
+        } catch (Exception e) {
+            // logging trace here because errors are expected when the pod is not running
+            log.trace "Unable to fetch logs for pod: $name", e
+            return null
+        }
+    }
```

Comment:
It should be considered to use a streaming approach, see


https://github.com/kubernetes-client/java/blob/master/examples/examples-release-latest/src/main/java/io/kubernetes/client/examples/LogsExample.java#L36-L49

Reply:
I didi tried it, but streamNamespacedPodLog() waits till the pod execution is completed, I will dig more

Reply:
I will move the logic to convert string to inputstream to this function so that getLogs return input stream

Reply:
I don't know if this is a limitation of java client or a bug, but I'm quite sure it's possible to stream the logs of a running pod. 

I've made a simple test running a pod like this 

```
kubectl run counter --image=busybox --command -- /bin/sh -c 'n=0; while true; do echo $((n++)); sleep 1;
```

Then I've used the [K8s client implemented in Nextflow](https://github.com/nextflow-io/nextflow/blob/1dc4e4e4f043695c413d47fdfda525159b87fd4f/modules/nextflow/src/main/groovy/nextflow/k8s/client/K8sClient.groovy#L571-L582), that's a bare simple http request over URLConnection. 

This snippet just stream the pod logs until the pod completes 

```
    def 'should get pod log' () {
        given:
        def settings = ['context': 'docker-desktop', name: 'default']
        def config = new K8sConfig(settings)
        def client = new K8sClient( config.getClient() )

        when:
        def logs = client.podLog([follow:'true'], 'counter')
        then:
        logs.text == 'hello\n'
    }
```